client/src/components/Dashboard/index.tsx

```diff
@@ -297,6 +331,21 @@ export default class Dashboard extends React.Component<IDashboardProps, IDashboa
       toolbarActions.push(
         (
           <span>
+            <SelectField
+              id="autorefresh"
+              label="Auto Refresh"
+              placeholder="0"
+              defaultValue={'None'}
+              position={SelectField.Positions.BELOW}
+              menuItems={['None', '30 Sec', '60 Sec', '2 Min', '5 Min', '30 Min']}
```

Comment:
Why not use an array of items with { name, value} - this way you can save the interval next to the text and don't need the `switch` in the `handleRefreshIntervalChange` method above